variables.tf

```diff
@@ -1,4 +1,6 @@
-
+###############################################
```

Comment:
> Also sort attributes for tags.

In the tags module?

Reply:
in this file, it just wasn't shown in the diff because it's part of the template project that you created without letting us review it :)

Reply:
If it's sort as in sorting the elements of the tags map alphabetically by key like
```
# take this
{ Name = "merge", Alias = "msimpson" }
# make it this
{Alias = "msimpson" , Name = "merge"}
```
that sounds like something the tags module should do

Reply:
Nah you just need to order the var attributes default, description, type. It's literally a line line change in variables.tf.

Reply:
I'm still confused. Can you add it as a suggestion?

Reply:
I can't because it's not in the files changed since you used the template. Here's the problem:

```
variable "tags" {
  description = "User-Defined tags"
  type        = map(string)
  default     = {}
}
```

Here's what it should look like:

```
variable "tags" {
  default     = {}
  description = "User-Defined tags"
  type        = map(string)
}
```

Reply:
I like this idea but this is something we shouldn't be doing manually and when we have a script for it we can just do a minor version bump

Reply:
It doesn't match eng standards and is a 1 line fix. You aren't gonna win me over on that, especially for a module that I'd like to think we won't have to touch for a while.